wadas/domain/db_model.py

```diff
@@ -30,32 +30,47 @@
 
 
 # Classes
+Base = declarative_base()
+
+
 class Camera(Base):
     __tablename__ = "cameras"
 
-    local_id = Column(Integer, primary_key=True, autoincrement=True)  # DB id and primary key
-    id = Column(String, nullable=False, unique=True)  # External (Camera class) unique identifier
+    local_id = Column(Integer, primary_key=True, autoincrement=True)  # DB id
+    id = Column(String, nullable=False, unique=True)  # Unique external ID
```

Comment:
id is a built-in identifier (like file, for example). Better to append a _ and use id_, instead.